binsync/extras/server/__init__.py

```diff
@@ -1,3 +1,28 @@
-def start_server():
-    print("Starting BinSync server...")
-    # TODO: implement server
\ No newline at end of file
+from flask import Flask
```

Comment:
`__init__` files in Python are not really for executing real code. They are for setting up the structure of the project, making imports simpler, and setting up imports for that module. Please move this code to an aptly named file.